### PR TITLE
Update screenshots section for iOS

### LINKED ISSUE #1889: Path of auto generated screenshots has changed (MSTG-STORAGE-9)
MSTG-STORAGE-9 states as path for saving background screenshots `/var/mobile/Containers/Data/Application/$APP_ID/Library/Caches/Snapshots/`.

Looks like it has changed or it's app related (tested on iOS 14.2). For me it's `/var/mobile/Containers/Data/Application/$APP_ID/Library/SplashBoard/Snapshots/sceneID:$APP_NAME-default/`

### PR DESCRIPTION
**Pull request by cpholguera**
- restructured whole test case to better match the new approach
- removed ObjC code

### CODE REVIEW

File: Document/0x06d-Testing-Data-Storage.md [hunk 1]
```diff
@@ -851,38 +851,16 @@ func applicationWillEnterForeground(_ application: UIApplication) {
...
-Navigate to an application screen that displays sensitive information, such as a username, an email address, or account details. Background the application by hitting the Home button on your iOS device. Connect to the iOS device and navigate to the following directory (which may be different for iOS versions below 14.2):
-
-`/var/mobile/Containers/Data/Application/$APP_ID/Library/SplashBoard/Snapshots/sceneID:$APP_NAME-default/`
-
-Older iOS versions may store the auto-generated screenshots under:
-
-`/var/mobile/Containers/Data/Application/$APP_ID/Library/Caches/Snapshots/`
-
-If the application caches the sensitive information in a screenshot, the app fails this test.
+Follow these steps using a jailbroken device or a non-jailbroken device after [repackaging the app with the Frida Gadget](0x06c-Reverse-Engineering-and-Tampering.md#dynamic-analysis-on-non-jailbroken-devices):
```
Comment (TheDauntless) [inline Document/0x06d-Testing-Data-Storage.md:856]: You could just verify this visually from the screen, without a jailbroken iphone and without Frida.

Maybe we could add a difference between 'validating the testcase' and 'collecting the evidence' ?

**Review by TheDauntless — COMMENTED** [review]
Seems ok, but one question.

File: Document/0x06d-Testing-Data-Storage.md (hunk 1 above)
Comment (cpholguera) [reply Document/0x06d-Testing-Data-Storage.md]: Agree & done ;)

**Comment by cpholguera** [comment]
Ready to go @TheDauntless :)

**Review by TheDauntless — APPROVED** [review]

**Pull request merged**